src/olympia/lib/settings_base.py

```diff
@@ -1924,6 +1924,13 @@ def get_raven_release():
 
 # Addon.average_daily_user count that forces dual sign-off for Blocklist Blocks
 DUAL_SIGNOFF_AVERAGE_DAILY_USERS_THRESHOLD = 100_000
+KINTO_API_URL = 'https://kinto.dev.mozaws.net/v1/'
+# The kinto test server needs accounts and setting up before using.
+KINTO_API_IS_TEST_SERVER = True
```

Comment:
> I'm not a big fan of this setting - the name doesn't really tell you what it does. 

alternative name suggestions welcome 😉 

> I'm also wondering if it would be better to default it to False to avoid accidents: would it actually manage to create anything if it was left enabled by mistake on stage or prod ?

It would depend what the credentials allowed, but it'd likely just fail instead.

Reply:
`KINTO_API_AUTO_REGENERATE_...` ugh not getting anywhere. I suppose it's fine.

Reply:
https://github.com/mozilla/addons-server/pull/13602/commits/18e4a93a8dbc566db72f5ef576119d531897bf96